src/index.js

```diff
@@ -23,11 +23,18 @@ const startMessage = `Conditional Tokens bot \`${version}\` was started.`;
 pushSlackMessage(startMessage);
 console.log(startMessage);
 
+const jobTime = process.env.JOB_GET_TRADE_MINUTES ? process.env.JOB_GET_TRADE_MINUTES : 5;
```

Comment:
You can create a CONFIG file, or maybe a constants file and then import it, so you don't to initialize this variable here

Reply:
Created issue

Reply:
https://github.com/gnosis/conditional-tokens-bot/issues/28